client/core/core.go

```diff
@@ -8403,9 +8403,9 @@ func parseCert(host string, certI interface{}, net dex.Network) ([]byte, error)
 	return nil, fmt.Errorf("not a valid certificate type %T", certI)
 }
 
-// walletDefinition gets the registered WalletDefinition for the asset and
+// WalletDefinition gets the registered WalletDefinition for the asset and
 // wallet type.
-func walletDefinition(assetID uint32, walletType string) (*asset.WalletDefinition, error) {
+func WalletDefinition(assetID uint32, walletType string) (*asset.WalletDefinition, error) {
```

Comment:
Kinda funky that this is a `core` functions.  It only uses the `client/asset` package internally, and it returns a `client/asset` type.